internal/output/output.go

```diff
@@ -223,3 +244,9 @@ func (o Output) printf(format string, a ...any) {
 	_, err := o.standardWriteCloser.Write([]byte(text))
 	o.errorCallback(err)
```

Comment:
printf should probably just call print(text)

Reply:
Fixed